saas_config.toml

```diff
@@ -48,4 +48,7 @@ domain = ""
 client_id = ""
 client_secret = ""
 redirect_uri = ""
-page_size = ""
\ No newline at end of file
+page_size = ""
+requester_email = ""
+identity_email = ""
+erasure_identity_email = ""
```

Comment:
since this is dynamically generated in the fixture (which i like), does it need to be in the `toml`? (specifically the `erasure_identity_email`). it seems like it could be misleading. but my understanding may be off.

Reply:
That's a good point, removed!